apis/cluster/services/definition.yaml

```diff
@@ -18,31 +18,61 @@ spec:
           spec:
             type: object
             properties:
+              deletionPolicy:
+                description: When the Composition is deleted, delete the AWS resources. Defaults to Delete
+                enum:
+                - Delete
+                - Orphan
+                type: string
+                default: Delete
+              gitops:
+                type: object
+                description: GitOps configure external gitops system for mcp controlplane
```

Comment:
That is not a clear statement for upbound/uxp agnostic platform ref. Should we generalize it?

Reply:
yes will do it